R/ReactantExtra/build_tarballs.jl

```diff
@@ -0,0 +1,222 @@
+using BinaryBuilder, Pkg
+using Base.BinaryPlatforms
+
+const YGGDRASIL_DIR = "../.."
+include(joinpath(YGGDRASIL_DIR, "fancy_toys.jl"))
+
+name = "ReactantExtra"
+repo = "https://github.com/EnzymeAD/Reactant.jl.git"
+version = v"0.0.1"
+
+sources = [
+   GitSource(repo, "e132afd886a6d8333de42e99f5c0806c40c66482"),
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd Reactant.jl/deps/ReactantExtra
+
+if [[ "${bb_full_target}" == x86_64-apple-darwin* ]]; then
+    # LLVM requires macOS SDK 10.14.
+    pushd $WORKSPACE/srcdir/MacOSX10.*.sdk
+    rm -rf /opt/${target}/${target}/sys-root/System
+    cp -ra usr/* "/opt/${target}/${target}/sys-root/usr/."
+    cp -ra System "/opt/${target}/${target}/sys-root/."
+    popd
+fi
+
+if command -v apk &> /dev/null
+then
+    apk add bazel --repository=http://dl-cdn.alpinelinux.org/alpine/edge/testing/
+    apk add py3-numpy py3-numpy-dev
+elif command -v brew &> /dev/null
+then
+    brew install bazel
```

Comment:
This seems weird for a BB script

Reply:
Yeah didn't realize that it didn't also run on an osx runner